src/baldr/graphtile.cc

```diff
@@ -450,32 +428,29 @@ GraphTile::FileSuffix(const GraphId& graphid, const std::string& fname_suffix, b
                           : TileHierarchy::levels()[graphid.level()];
 
   // figure out how many digits
-  auto max_id = level.tiles.ncolumns() * level.tiles.nrows() - 1;
-  size_t max_length = static_cast<size_t>(std::log10(std::max(1, max_id))) + 1;
-  const size_t remainder = max_length % 3;
-  if (remainder) {
-    max_length += 3 - remainder;
-  }
+  auto get_num_digits = [&level]() {
+    auto max_id = level.tiles.ncolumns() * level.tiles.nrows() - 1;
+    size_t max_length = static_cast<size_t>(std::log10(std::max(1, max_id))) + 1;
+    const size_t remainder = max_length % 3;
+    if (remainder) {
+      max_length += 3 - remainder;
+    }
+    return max_length;
+  };
+  const size_t num_digits = get_num_digits();
```

Comment:
Why is this a separate lambda rather than just having the code in-line?

Reply:
Idea here is to limit scope of temporary variables like `remainder` and `max_id` and make sure they are used only for calculation of string length. But ok, I will remove lambda.

Reply:
ah; could just be `{}` scope. alternatively, you could use an immediately invoked lambda, i.e. don't give it a name and instead call it immediately.

Reply:
Lambda removed.